assets/minecraft/models/item/diamond_sword.json

```diff
@@ -91,6 +91,8 @@
 		{ "predicate": {"custom_model_data": 85}, "model": "item/mob/titanjaw/titanjaw_hit"},
 		{ "predicate": {"custom_model_data": 86}, "model": "item/mob/stingerhead/stingerhead"},
 		{ "predicate": {"custom_model_data": 87}, "model": "item/mob/stingerhead/stingerhead"},
-		{ "predicate": {"custom_model_data": 88}, "model": "item/mob/stingerhead/stingerhead"}
+		{ "predicate": {"custom_model_data": 88}, "model": "item/mob/stingerhead/stingerhead"},
+		{ "predicate": {"custom_model_data": 89}, "model": "item/mob/hisomukuro/hisomukuro"},
+		{ "predicate": {"custom_model_data": 90}, "model": "item/mob/hisomukuro/hisomukuro_hit"}
```

Comment:
the general pattern we use is
- idle
- moving
- hit

so please leave one number in between these two for the moving one, just set it to the same as the idle until moving animation comes around.